pkgs/by-name/rk/rkbin/package.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rkbin";
+  version = "2023.07.26-${gitShort}";
+
+  gitCommit = "b4558da0860ca48bf1a571dd33ccba580b9abe23";
```

Comment:
Can you move these two into an outer let binding, so they're not bleeding as env vars into the build itself? Or do these need to be set in the build? If so, a comment on top would be good.

Reply:
Sure, I can move them in a let binding outside since those don't actually need to be in the build itself.